### PR TITLE
Updating the banner

### PR DESCRIPTION
**Pull request by sfshaza2**
Don't publish until Wednesday at 10am.

### CODE REVIEW

**Comment by domesticmouse** [comment]
Marking as draft to prevent accidental submission

**Comment by sfshaza2** [comment]
Hmmm. I got this link from Kelvin. Can you check with him?

File: src/_includes/banner.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% comment %}Drop the div .alert classes to revert to the default banner styling.{% endcomment -%}
 <div class="banner">
   <p class="banner__text">
-    Join Dart and Flutter at Google I/O 2022 live from Shoreline Amphitheatre and online May 11-12. <a href="https://io.google/2022/?utm_source=dart&utm_medium=embedded_marketing&utm_campaign=hpp_svd&utm_content=">Register now</a><br>
+    <a href="https://io.google/2022/?utm_source=dart&utm_medium=embedded_marketing&utm_campaign=hpp_svd&utm_content=">Google I/O is happening now. Join us.</a><br>
```
[the remark this answers is not in the document]
Comment (parlough) [reply src/_includes/banner.html:4]: Maybe for day 2? If we're publishing this at 10 AM that's when the main keynotes start which wouldn't be visible from the Flutter page.

Comment (parlough) [inline src/_includes/banner.html:4]: Maybe remove the final period to match our other banner links:

```suggestion
    <a href="https://io.google/2022/?utm_source=dart&utm_medium=embedded_marketing&utm_campaign=hpp_svd&utm_content=">Google I/O is happening now. Join us</a><br>
```

**Review by parlough — APPROVED** [review]

**Pull request merged**